README.md

```diff
@@ -1,47 +1,153 @@
+* [Docker Compose](https://docs.docker.com/compose/install/) (v1.25.0+)
+* [Node.js](https://nodejs.org/en/download/) (v10.0+)
 
-Download and unzip [package](https://github.com/dashevo/mn-bootstrap/archive/master.zip).
+### Distribution package 
+
+```bash
+$ git clone -b master https://github.com/dashevo/mn-bootstrap.git
+$ cd mn-bootstrap
+$ npm install # optional: install CLI dependencies
+$ sudo npm link # optional: link CLI for system-wide execution
+```
 
 ## Usage
 
-Package contains Docker Compose file and configuration presets.
+The package contains a CLI, Docker Compose files and configuration presets.
 
-### Configure
+### Configuration presets
 
-Package contains several configuration presets:
  - Local - standalone masternode for local development
  - Evonet - masternode with Evonet configuration
  - Testnet - masternode with testnet configuration
 
-There are two ways to apply a present:
- 1. Rename corresponding dotenv file (i.e. `.env.local`) to `.env`
- 2. Add `--env-file` option to `docker-compose` command
+### CLI
+
+The CLI can be used to perform routine tasks. Invoke the CLI with `mn` if linked during installation, or with `node bin/mn` if not linked. To list available commands, either run `mn` with no parameters or execute `mn help`. To list the help on any command just execute the command, followed by the `--help` option
+
+### Start node
+
+The `start` command is used to start a node with a specified configuration preset.
+
+```
+USAGE
+  $ mn start PRESET EXTERNAL-IP CORE-P2P-PORT
+ARGUMENTS
+  PRESET         (local|testnet|evonet) preset to use
+  EXTERNAL-IP    masternode external IP
+  CORE-P2P-PORT  Core P2P port
+OPTIONS
+  -f, --full-node                                  start as full node
+  -p, --operator-private-key=operator-private-key  operator private key
+```
+
+To start a masternode for Evonet:
+
+```bash
+$ mn start evonet 1.2.3.4 20001 -p 2058cd87116ee8492ae0db5d4f8050218588701636197cfcd124dcae8986d514
+```
+
+To start a full node for Evonet:
+
+```bash
+$ mn start evonet 1.2.3.4 19999 -f
+```
+
+### Stop node
+
+The `stop` command is used to stop a running node.
+
+```
+USAGE
+  $ mn stop PRESET
+ARGUMENTS
+  PRESET  (local|testnet|evonet) preset to use
+```
+
+To stop an Evonet node:
 
-### Start
+```bash
+$ mn stop evonet
+```
+
+### Register masternode
+
+The `register` command creates a collateral funding transaction and then uses it to register a masternode on the specified network. It does not configure or start a masternode on the host.
+
+#### Funding collateral
+
+Before registering the masternode, you must send more than 1000 Dash to an address on the network you intend to use. 1000 Dash is used for the collateral transaction, and the remainder will be used for transaction fees. Make sure you have access to the private key for this address, since you will need to provide it in the next step. If using Dash Core, you can get the private key for a given address using the following command:
+
```

Comment:
Don't we need to describe `mn wallet generate-to-address` here as an option for the local preset?

Reply:
Done, please review :)

Reply:
Looks good! 👍